packages/microservice-util-lib/tsconfig.lib.json

```diff
@@ -1,8 +1,7 @@
 {
     "extends": "./tsconfig.json",
     "compilerOptions": {
-        "rootDir": "src",
-        "outDir": "dist",
+        "outDir": "../../dist/out-tsc",
```

Comment:
What caused this change?

Reply:
I copy this as it from a newly generated package by @nx/js plugin. Now looking back, I think we are safe to remove it as `outDir` is now handled by the options for `@nx/js:tsc` executor.

Reply:
If that works let's remove it